### ISSUE TITLE
Meander Harmony and Bass section gate output no longer polyphonic?

### ISSUE THREAD

**Issue by Ghalebor** [knchaffin/Meander#11]
Hi Ken. I was a bit surprised that in the latest version Meander lost the polyphonic gate outputs in the Harmony and Bass section. So it has to be? Is this just a small error in the code? I hope this is only a bug that can be fixed (in my opinion, it completely changes the playability of your module, especially since the 1V / Oct outputs in both sections are polyphonic). From what I remember in the previous version, both gate exits were polyphonic. Regards. Mateusz Jędrzejewski

**Comment by knchaffin** [comment]
Hi Mateusz.  The Meander gate outputs have always been monophonic.   Meander has no code to deal with polyphonic gates   When a port is created, by default the SDK creates it as monophonic and then via code port.SetChannels(n) is used to set the port number of channels to "n".   Meander uses polyphony on the harmony and bass 1V/oct CV output ports.  My assumption is that if the monophonic gate output cable is hooked up to an ADSR, for example, the single channel voltage gets duplicated to all ADSR channels, if the the ADSR is polyphonic.  At least that seems to be the Rack SDK standard.

No one has mentioned this before and I haven't even given it much thought.  I think I did post a comment in my development thread that I had just realized that gates could be polyphonic.  Just glancing around, I have not found a Rack module with polyphonic gate inputs or outputs.  If you point me towards one, I could look into this deeper to try to get a better feel for the issue.

Also let me know if I am misunderstanding your issue.

Thanks and regards, Ken Chaffin
